lte/gateway/python/magma/pipelined/app/enforcement_stats.py

```diff
@@ -83,10 +83,7 @@ def __init__(self, *args, **kwargs):
         self._clean_restart = kwargs['config']['clean_restart']
         self._redis_enabled = kwargs['config'].get('redis_enabled', False)
         # Store last usage excluding deleted flows for calculating deltas
-        if self._redis_enabled:
-            self.last_usage_for_delta = UsageDeltaDict()
-        else:
-            self.last_usage_for_delta = {}
+        self.last_usage_for_delta = defaultdict(RuleRecord)
         self.failed_usage = {}  # Store failed usage to retry rpc to sessiond
         self._unmatched_bytes = 0  # Store bytes matched by default rule if any
```

Comment:
We could clear enforcement table flow stats using OFPFF_RESET_COUNTS on stat up, by re-adding existing flows to OVS.

https://github.com/faucetsdn/ryu/blob/a394673993468ad3ee6cd784eb0d52aab967a8b3/ryu/ofproto/ofproto_v1_4_parser.py#L4353

Reply:
That is quite nice, I do have 1 concern though. We're aiming for flat rate reporting in near term future, if I wipe stats on restart then it won't work well with flat rate

Reply:
right. flat rate reporting is better solution. this is more of stopgap.